Makefile

```diff
@@ -98,8 +112,8 @@ style/fix: ## Fix code style
 
 .PHONY: migrations
 migrations: ## Execute database migrations
-	docker exec --user=$$(id -u):$$(id -g) $(DOCKER_CONTAINER) php bin/console doctrine:migrations:migrate --no-interaction
+	docker exec --user=$$(id -u):$$(id -g) $(DOCKER_CONTAINER) php bin/console doctrine:migrations:migrate --no-interaction --quiet --allow-no-migration
 
 .PHONY: migrations-prev
 migrations-prev: ## Rollback the last migration
-	docker exec --user=$
\ No newline at end of file
+	docker exec --user=$$(id -u):$$(id -g) $(DOCKER_CONTAINER) php bin/console doctrine:migrations:migrate prev --no-interaction --quiet --allow-no-migration
```

Comment:
@yebrai Here is the fix :smiley: If you can please check this PR as soon as you can :wink:

Reply:
Ok :)